src/Smart.CFA.Pages/Pages/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - Smart.CFA.Pages</title>
```

Comment:
Do we need the string ` - Smart.CFA.Pages` in the title?

Reply:
Hmm the whole view is just a template, but it allowed me to see nginx make the redirection. So it was good while it lasted but will be removed altogether, with all the other view in the current project